src/Skinner.ts

```diff
@@ -14,12 +14,20 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-class Skinner {
-    constructor() {
-        this.components = null;
-    }
+import React from "react";
+
+export interface IComponents {
```

Comment:
I am quite unsure about the typing in this file. What type is a `component` here?

Reply:
it's a `React.Component` for lack of better types